packages/story-editor/src/elements/video/display.js

```diff
@@ -117,20 +118,13 @@ function VideoDisplay({ previewMode, box: { width, height }, element }) {
           data-testid="videoElement"
           data-leaf-element="true"
         >
-          {url && <source src={url} type={resource.mimeType} />}
-          {tracks &&
-            tracks.map(({ srclang, label, track: src, id: key }, i) => (
-              <track
-                srcLang={srclang}
-                label={label}
-                // Hides the track from the user.
-                // Displaying happens in MediaCaptionsLayer instead.
-                kind="metadata"
-                src={src}
-                key={key}
-                default={i === 0}
-              />
-            ))}
+          {
+            url && <source src={url} type={resource.mimeType} />
+
+            // Hides the track from the user
+            // Displaying happens in MediaCaptionsLayer instead.
+          }
+          <Captions tracks={tracks} />
```

Comment:
This is where you need to actually set `kind="metadata"`

The other components don't need that.

`defaultKind` doesn't really achieve that